### PR TITLE
sysvar_cache: missing cache checks

### PR DESCRIPTION
**Pull request by jumpsiegel**

### CODE REVIEW

File: src/flamenco/runtime/sysvar/fd_sysvar_slot_history.c [hunk 1]
```diff
@@ -120,7 +120,7 @@ fd_sysvar_slot_history_update( fd_exec_slot_ctx_t * slot_ctx ) {
 
 int
 fd_sysvar_slot_history_read( fd_exec_slot_ctx_t * slot_ctx,
```
Comment (mjain-jump) [inline src/flamenco/runtime/sysvar/fd_sysvar_slot_history.c:122]: do we need the extra query + check here too?

Comment (jumpsiegel) [reply src/flamenco/runtime/sysvar/fd_sysvar_slot_history.c:122]: This is not in the cache

**Review by topointon-jump — APPROVED** [review]
Do you mind adding the fixture? 🙏

**Comment by jumpsiegel** [comment]
was going to add the fixture once all the tests passed..

**Review by mjain-jump — DISMISSED** [review]
lgtm

**Review by topointon-jump — DISMISSED** [review]

**Review by topointon-jump — APPROVED** [review]

**Pull request merged**